common.blocks/issue/issue.bemtree.js

```diff
@@ -12,35 +18,65 @@ block('issue').content()(function() {
-            content: issue.labels.map(function(label) {
-                return {
-                    elem: 'label',
-                    attrs: { style: 'border: 1px solid #' + label.color },
+            elem: 'container',
+            content: [
+                {
+                    elem: 'title',
                     content: {
                         block: 'link',
                         mix: { block: 'page', elem: 'link' },
-                        url: '/?labels=' + label.name,
-                        attrs: { style: 'color: #' + label.color },
-                        content: label.name
+                        url: '/' + issue.number + '/',
+                        content: issue.title
                     }
-                };
-            })
-        } : '',
-        {
-            elem: 'content',
-            content: issue.html
+                },
+                issue.labels.length ? {
+                    elem: 'labels',
+                    content: issue.labels.map(function(label) {
+                        return {
+                            elem: 'label',
+                            attrs: { style: 'border: 1px solid #' + label.color },
+                            content: {
+                                block: 'link',
+                                mix: { block: 'page', elem: 'link' },
+                                url: '/?labels=' + label.name,
+                                attrs: { style: 'color: #' + label.color },
+                                content: label.name
+                            }
+                        };
+                    })
+                } : '',
+                {
+                    elem: 'content',
+                    content: issue.html
+                },
+                {
+                    elem: 'spiner-overlay',
+                    content: {
+                        elem: 'loader',
+                        content: {
+                            block: 'spin',
+                            mix: { block: 'content', elem: 'spin' },
+                            mods: { theme: 'islands', size: 'xl', visible: false }
+                        }
+                    } 
+                }
+            ]
         },
-        this.data.url.pathname === '/' ? {
+        (issue['state'] === 'closed') ? {
+            block: 'button',
+            tag: 'div',
+            mods: { theme: 'islands', size: 'm', type: 'submit', view: 'pseudo', disabled: true },
+            mix: { block: 'issue', elem: 'has_answer_label' },
+            text: (isIssueAuthor) ? i18n(this.block, 'iGotAnswer') : i18n(this.block, 'hasAnswer')
+        }: '',
+        (isIssueAuthor && isIssuePage) ? {
+            block: 'button',
+            mods: { theme: 'islands', size: 'm', type: 'link', view: 'action' },
+            mix: { block: 'issue', elem: 'close-button' },
+            url: '/api/set_issue_state/' + issue.number + ((issue['state'] === 'open') ? '/closed/' : '/open/'),
```

Comment:
I'd rather cache state as a flag `var isOpen = issue.state === 'open'` and reuse everywhere